packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -1074,6 +1074,15 @@ new ec2.Instance(this, 'Instance4', {
   }),
 });
 
+// AWS Linux 2023
+new ec2.Instance(this, 'Instance4', {
+  vpc,
+  instanceType,
+  machineImage: ec2.MachineImage.latestAmazonLinux({
+    generation: ec2.AmazonLinuxGeneration.AMAZON_LINUX_2023,
+  }),
+});
+
```

Comment:
Please remove the example for "AWS Linux 2022" as it is unnecessary, and update the comment for "AWS Linux 2023" to say "Amazon Linux 2023" instead.